src/com/googlecode/jmxtrans/model/naming/KeyUtils.java

```diff
@@ -254,7 +254,10 @@ private static String getTypeNameValuesSeparator(Query query) {
 		String[] result;
 		String[] keys = typeNameToken.split("=", 2);
 
-		if (keys.length == 2) {
+		for (int i=0; i<keys.length; i++) {
+		}
```

Comment:
How embarrassing.. I needed to do more than just delete all printlns /me promises not to commit while sitting in Starbucks in future!